### PR TITLE
Accessibility implementations

### PR DESCRIPTION
**Pull request by fibini**
**Title** : No issues

**Image text alternatives** : https://github.com/fibini/Portfolio-project/compare/Accessibility-Implementations?expand=1#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051R77-R102

**Text Headings**: No issues

**Color contrast**: No issues

**Resize**: https://github.com/fibini/Portfolio-project/compare/Accessibility-Implementations?expand=1#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051R18-R21 and  https://github.com/fibini/Portfolio-project/compare/Accessibility-Implementations?expand=1#diff-b78be019f1dc6d57753ea900c3805b114cd53ab7c0db836cc081836df1b99b7aR787-R793

**Interaction** : No issues

**Moving content**: No issues

**Multimedia**: No issues

**Basic structure of page**: No issues

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -5,7 +5,7 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <link rel="stylesheet" href="style.css" />
-    <title>Document</title>
+    <title>Portfolio-Project</title>
```
Comment (Ismailco) [inline index.html:8]: - [ ] The page title should briefly describe the content of the page, please consider changing it to a meaningful title. For example:
  -  Fabien Portfolio

File: index.html [hunk 2]
```diff
@@ -5,7 +5,7 @@
...
   </head>
   <body>
```
Comment (Ismailco) [inline index.html:10]: - [ ] Heading levels should have a meaningful hierarchy, please consider using the ```<h1>``` tag only one time preferably for the page header, and use ```<h2>``` to ```<h6>``` in descending way, so you can respect the hierarchy of the ```hx``` tags

**Review by Ismailco — CHANGES_REQUESTED** [review]
### Hi @fibini,

Good job so far 👍🏼 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✔️ No linter errors
✔️ Project requirements
✔️ Work documented professionally

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by fibini** [comment]
Please can you let me know if my partner also needs to submit this pull request or does he have to submit a different pull request?

File: index.html [hunk 3]
```diff
@@ -5,26 +5,26 @@
...
-          <h4 class="adj">About</h4>
-          <h4 class="adj">Contact</h4>
-          <img class="letter" src="./img/Union1.png" alt="mail image" />
+          <a class="adj" href="">Portfolio</a>
+          <a class="adj" href="">About</a>
+          <a class="adj" href="">Contact</a>
+          <a href=""><img class="letter" src="./img/Union1.png" alt="mail image" /></a>
         </nav>
-      </header>
+      </section>
```
Comment (phelian23) [inline index.html:23]: - [ ] Good job working on the changes requested by the previous reviewer. However, I noticed you changed the toolbar section wrapper from a `<header>` tag to a `<section>` tag. For semantic purposes, kindly make the wrapper a `<header>` tag. You can read more about the use of a `<header>` tag [here](https://www.w3schools.com/tags/tag_header.asp)

**Review by phelian23 — CHANGES_REQUESTED** [review]
Hi @fibini, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 3 above)
Comment (fibini) [reply index.html:23]: so can I have 2 header tags then? @phelian23

Comment (fibini) [reply index.html]: ok so I can have more than 1 header, once it is not in a footer, address or in another header, so there should be no problems now

**Review by Omar-Muhamad — APPROVED** [review]
# Approved ✔
### Hi Team,

Great job, Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

thumbs-up-agree


## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- Github flow is used correctly. ✔
- All linters checks are ok. ✔

## (Changes Required) Aspects to improve: ♻️

- N/A

## [Optional] suggestions:

- N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**